cassovary-core/src/test/scala/com/twitter/cassovary/algorithms/TriangleCountSpec.scala

```diff
@@ -23,6 +25,10 @@ class TriangleCountSpec extends WordSpec with Matchers {
     (sums._1 / repetitions, sums._2 / repetitions)
   }
 
+  val random = new Random()
+
+  def rng(): Random = {random.setSeed(1); random}
```

Comment:
prefer not reusing 'val random' across tests. so create new object everytime in rng()